fides.toml

```diff
@@ -0,0 +1,14 @@
+[database]
+SERVER="db"
+USER="postgres"
+PASSWORD="216f4b49bea5da4f84f05288258471852c3e325cd336821097e1e65ff92b528a"
+DB="app"
+TEST_DB="test"
+
+[security]
+APP_ENCRYPTION_KEY="OLMkv91j8DHiDAULnK5Lxx3kSCov30b3"
+CORS_ORIGINS=["http://localhost", "http://localhost:8080", "http://localhost:3000", "http://localhost:3001"]
+ENCODING="UTF-8"
+OAUTH_ROOT_CLIENT_ID="fidesadmin"
+OAUTH_ROOT_CLIENT_SECRET="fidesadminsecret"
+DRP_JWT_SECRET="secret"
```

Comment:
This is interesting. I guess there are use cases for running Fideslib in isolation, like testing, where you'd need the infrastructure spun up within this repo. I was thinking these things should be override-able in the importing repos too.

Reply:
This file is for testing purposes. When the package gets pip installed it will pickup which ever file that repo has, i.e. fidesops, so the installer of the package will have control over these values.

Reply:
This path can be configured in the environment variables as assuming my comments on that part holds true this should be ok. We can also add `repository-root/.fides` to the default search path, and add `fidesctl.toml` to the default files searched for.

That said, once we think everything is good with this PR it would be a good idea to do a beat release to pypi where we can fully test it in all the different repos because the whole config part is complicated.